tests/src/test/scala/com/nvidia/spark/rapids/ProjectExprSuite.scala

```diff
@@ -54,6 +54,11 @@ class ProjectExprSuite extends SparkQueryCompareTestSuite {
         lit(BigDecimal(-2.12314e-8)),
         lit(Array(1, 2, 3, 4, 5)), lit(Array(1.2, 3.4, 5.6)),
         lit(Array("a", "b", null, "")),
+        lit(Array(Array(1, 2), null, Array(3, 4))),
```

Comment:
The decription of this pr is only "Array" , I guess you mean array of array, and array of structs.

Reply:
I thought array type also covered array of array, and array of structs, is it ?

Reply:
I mean when I first read the description, my feeling is "before this PR, array literal is not supported and this PR starts to support it."

Reply:
I am happy with doing more than is promised, especially if it is simple to add in. Please do not remove array of structs even if it is beyond the requirements.

Reply:
Updated